webdriver-spec.html

```diff
@@ -829,7 +829,7 @@
 
  <li><p>If <var>data</var> is not null, let <var>response</var>’s
   <a>body</a> be a JSON <a>Object</a> with a key <code>value</code>
-  set to the <a>JSON serialisation</a> of <var>data</var>.
+  set to the <a>JSON clone</a> of <var>data</var>.
```

Comment:
This should stay the same, I just need to get around to define _JSON serialisation_.